dist/configure

```diff
@@ -416,6 +439,9 @@ if detect_x64; then
     echo "... $build_target supports our inline ASM"
     compile_inline_asm=true
   fi
+  echo "... $build_target supports AES/PCLMUL instruction set extensions"
+  compile_aesni=true
+  echo "CFLAGS_AESNI = -maes -msse4.1" >> Makefile.config
```

Comment:
do all toolchains know these build flags? I suppose so... @karthikbhargavan do we support ancient redhats still for Mozilla? maybe irrelevant for the purposes of our own CI

Reply:
Let's just make sure it runs on our CI and on hacl packages. NSS has its own constraints and we will deal with them over there.